### PR TITLE
Add mutli-set elements (`Þ∨` - difference, `Þ∩` - intersection, `Þ∪` - union, `Þ⊍` - xor)

### LINKED ISSUE #1066: Multiset builtins
Specifically, multiset intersection, xor, and union.

### PR DESCRIPTION
**Pull request by Lyxal**
Closes #1066

### CODE REVIEW

**Comment by Lyxal** [comment]
Tested against the results of Jelly.

File: vyxal/elements.py [hunk 1]
```diff
@@ -3023,6 +3023,60 @@ def multiply(lhs, rhs, ctx):
...
+
+    lhs = iterable(lhs, ctx=ctx)
+    rhs = iterable(rhs, ctx=ctx)
+    res = []
+
+    for item in rhs:
+        if item in lhs:
+            res.append(item)
+
+    return res
```
Comment (chunkybanana) [inline vyxal/elements.py:3055]: This doesn't seem to handle the multiset part - it looks like passing `[1, 1, 1]` and `[1]` will return `[1, 1, 1]` where it should return `[1]`.

File: vyxal/LazyList.py [hunk 2]
```diff
@@ -267,3 +267,16 @@ def reversed(self):
         self.generated += list(itertools.tee(self.raw_object)[-1])
         for item in self.generated[::-1]:
             yield item
+
+    def remove(self, value):
```
Comment (cgccuser) [inline vyxal/LazyList.py:271]: Weren't we were planning on making `LazyList`s immutable? This mutates the generated list.

**Review by cgccuser — CHANGES_REQUESTED** [review]

**Review by chunkybanana — CHANGES_REQUESTED** [review]

**Review by cgccuser — APPROVED** [review]

**Review by SE-Nitro — APPROVED** [review]
I don't want to merge it yet, I might have missed something

**Review by chunkybanana — APPROVED** [review]

**Pull request merged**